plugins/reach/lib/model/VendorCatalogItem.php

```diff
@@ -229,5 +244,43 @@ public function requiresEntryReady()
 		return true;
 	}
 
+	public function setStage($v)
+	{
+		$this->putInCustomData(self::CUSTOM_DATA_STAGE, $v);
+	}
+
+	public function getStage()
+	{
+		return $this->getFromCustomData(self::CUSTOM_DATA_STAGE, null, VendorCatalogItemStage::PRODUCTION);
+	}
+
+	public function setContract($v)
+	{
+		$this->putInCustomData(self::CUSTOM_DATA_CONTRACT, $v);
+	}
+
+	public function getContract()
+	{
+		return $this->getFromCustomData(self::CUSTOM_DATA_CONTRACT, null, VendorCatalogItemStage::PRODUCTION);
```

Comment:
why contract, createdBy and notes going by deafult to production?
looks like copy past mistake from stage